mautic/mautic
# Add phpunit-bridge to the project.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ x ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ x ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 



#### Description:
Add phpunit-bridge to check for deprecations during development process.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Enable deprecations by changing `phpunit.xml.dist` env value to `<env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=100000"/>`
3. Run the test suite
4. Check deprecations are displayed.
